### ISSUE TITLE
Syntax highlighting stops after destructuring when defining each loop.

### ISSUE THREAD

**Issue by morleytatro** [UnwrittenFun/svelte-vscode#11]
When I use the following to define a loop, syntax highlighting stops right after the destructuring. Without destructuring, the issue goes away.
```
{#each sortTypes as { name, text } (name)}<button
  class="nav-btn{sortType === name ? ' active' : ''}"
  on:click="setSort(name)"
>{text}</button>{/each}
```

**Comment by jamesbirtles** [comment]
Fixed in 0.3.1

**Comment by morleytatro** [comment]
Looks great. Thanks James!
